activation/activation.go

```diff
@@ -976,8 +1008,13 @@ func findFullyValidHighTickAtx(
 		return false
 	})
 
-	if found != nil {
-		return *found, nil
+	if ctxErr != nil {
+		return emptyAtx, ctxErr
 	}
```

Comment:
NIT: the local variables aren't need

```suggestion
	if ctx.Err() != nil {
		return types.EmptyATX, ctx.Err()
	}
```